webapp/components/CategoriesSelect/CategoriesSelect.spec.js

```diff
@@ -1,9 +1,7 @@
-import { mount, createLocalVue } from '@vue/test-utils'
+import { mount } from '@vue/test-utils'
 import CategoriesSelect from './CategoriesSelect'
-import Styleguide from '@human-connection/styleguide'
 
-const localVue = createLocalVue()
-localVue.use(Styleguide)
+const localVue = global.localVue
```

Comment:
Do we clean `localVue` at some point?

Reply:
We don't. But the `setupFiles` create a new instance of `localVue` before running each test file and the `global` variable should only be available in the `test` scope, no?